shared/wallets/wallet/header.js

```diff
@@ -123,33 +116,31 @@ class _DropdownButton extends React.PureComponent<DropdownProps & OverlayParentP
 
   render() {
     return (
-      <ClickableBox onClick={this.props.toggleShowingMenu} ref={this.props.setAttachmentRef}>
-        <Box2 direction="horizontal" fullWidth={true} gap="xsmall">
-          <Button onClick={null} type="Secondary" label="..." />
-        </Box2>
-        <FloatingMenu
+      <Kb.ClickableBox onClick={this.props.toggleShowingMenu} ref={this.props.setAttachmentRef}>
+        <Kb.Box2 direction="horizontal" fullWidth={true} gap="xsmall">
+          <Kb.Button onClick={null} type="Secondary">
```

Comment:
Left/right paddings on this button should be 16px on desktop and 12px on mobile, like on the profile one. Otherwise the button looks too wide. Ellipsis icon should be 16px (the default) on desktop and 22px on mobile.